crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -450,6 +450,10 @@ define_tedge_config! {
             /// Enable auto registration feature
             #[tedge_config(example = "true", default(value = true))]
             auto_register: bool,
+
+            /// Determines if entity state on tedge should be sent to cloud on startup
+            #[tedge_config(example = "true", default(value = true))]
+            sync_on_startup: bool,
```

Comment:
@albinsuresh why are you no happy with @Ruadhri17 proposal?

I'm okay with `sync_on_startup`, `sync_on_start` and `clean_start`, but with none of the alternatives. These alternatives will only puzzle the users.

Reply:
That was on a request from @reubenmiller because the word `sync` might imply two-way synchronization. So, we wanted to change it to something that implies unidirectional (device to cloud). I finally settled on `clean_start` as it's short and implies exactly what the code does.